CONTRIBUTING.md

```diff
@@ -177,16 +177,27 @@ $ npm run proxy
 
 Once Prism is listening, you can issue http requests to the proxy port, in this
 case `http://127.0.0.1:4010` using the client of your choice. The example below uses
-[httpie](https://httpie.io/docs#main-features), a user friendly client.
+[httpie](https://httpie.io/docs#main-features), a user friendly client. Note: The API key used to authenticate this request is hardcoded into the example here because the example address whose ID we're using can only be accessed with this particular key (which is from the docs).
```

Comment:
It just occurred to me that this approach won't work, because that key is tied to docs.lob.com, which will be going away! We won't have a test key hard-coded in the new developer docs. I think you can still give the example, just redact the sensitive bits.